docs/conf/conf.py

```diff
@@ -0,0 +1,82 @@
+# -- Project information -------------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "CLP"
+# NOTE: We don't include a period after "Inc" since the theme adds one already.
+copyright = "2021-2024 YScope Inc"
+
+# -- General configuration -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "myst_parser",
+    "sphinx_copybutton",
+    "sphinx_design",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.viewcode",
```

Comment:
technically speaking `_` comes after `.`

Reply:
Based on our discussion offline, the tool I was using to sort (VSCode) seems to treat `sphinx_design` as one word but `sphinx.ext` as two words, and it chooses to sort single words before multiple words. Our conclusion was to rely on the tool rather than ASCII ordering.